app/src/test/java/com/example/workoutapp/WorkoutUnitTest.kt

```diff
@@ -0,0 +1,167 @@
+package com.example.workoutapp
```

Comment:
these unit tests failed for me because of lack to implement necessary classes for it